### PR TITLE
Activity bar Icon should look similar to vscode

### PR DESCRIPTION
**Pull request by daimor**
#### What it does

Solves the issue with truncated icons on Activity Bar

Now the icon should look better
image
image

#### How to test

Install extension daimor.vscode-objectscript, keywords: intersystems, objectscript
Check the icon look

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: packages/debug/src/browser/style/index.css [hunk 1]
```diff
@@ -156,12 +156,12 @@
 }
 
 .debug-tab-icon {
-    -webkit-mask: url('debug-dark.svg') 50%;
+    -webkit-mask: url('debug-dark.svg') 50% 50% / var(--theia-private-sidebar-icon-size) no-repeat;
```
Comment (akosyakov) [inline packages/debug/src/browser/style/index.css:159]: Could you elaborate on this change?
Should we align mask as well?
What if the widget is moved to the main or bottom area? Does icon still look good?

Comment (daimor) [reply packages/debug/src/browser/style/index.css:159]: I've just copied that style from VSCode. Previously it set a fixed size
```
width: var(--theia-private-sidebar-icon-size);
height: var(--theia-private-sidebar-icon-size);
```
and it was just about 24px, while toolbar size is 50px, so I replaced it with bigger size, and removed padding, which I suppose used to keep icon in the center.
```
  width: var(--theia-private-sidebar-tab-width);
  height: var(--theia-private-sidebar-tab-width);
```
So, now it supposed to be placed in the center with the `-webkit-mask`, and have a size as previously, but it now affects only for width while height as auto.
I'll extend the to `mask` as well.  Just checking how it looks in Firefox.

> What if the widget is moved to the main or bottom area? Does icon still look good?
I don't know how to do it, if you would give me info, how to do it, I will check

Comment (daimor) [reply packages/debug/src/browser/style/index.css:159]: Main 
image
Bottom
image
RIght
image

**Review by akosyakov — APPROVED** [review]
code wise looks reasonable

@vince-fugnitto Could you test please and merge?

**Comment by daimor** [comment]
> I attempted to move the tabbar in the main/bottom area and it had problems rendering (truncated icon):
> 
> main-tab

fixed

**Comment by daimor** [comment]
Hi, when it can be merged? just would like to start using it already with the next releases.

**Comment by vince-fugnitto** [comment]
> Hi, when it can be merged? just would like to start using it already with the next releases.

It should be fine to merge, it looks like no one else wanted to try or has any objections so I'll merge :)

**Comment by kittaakos** [comment]
@daimor, your changes break the icons for downstream projects. If you're willing to fix it, please let us know. Otherwise, I am going to revert this PR. Thanks for your understanding.

Before:
Screen Shot 2020-09-15 at 10 16 52

After:
Screen Shot 2020-09-15 at 10 17 16

**Comment by daimor** [comment]
Sure, I can do it, could you help me how to reproduce it?

**Comment by kittaakos** [comment]
Thank you so much for your prompt reply.

> could you help me how to reproduce it?

Good point. I have to figure out how to set up an example with the defect. I'll keep you posted.

**Comment by kittaakos** [comment]
I have created a [branch](https://github.com/eclipse-theia/theia/tree/GH-8426--follow-up) with the example. ([Here](https://github.com/eclipse-theia/theia/compare/GH-8426--follow-up) is the diff.)

Steps to reproduce:
 - Open the `Sample Unclosable View`.

screencast 2020-09-15 10-51-46

**Comment by daimor** [comment]
It the first look, `examples/api-samples/src/browser/style/index.css` should have this
```
library-tab-icon {
    -webkit-mask: url('../icons/library-tab-icon.svg') 50% 50% / var(--theia-private-sidebar-icon-size) no-repeat;
    mask: url('../icons/library-tab-icon.svg') 50% 50% / var(--theia-private-sidebar-icon-size) no-repeat;
}
``

**Comment by kittaakos** [comment]
I did expect a breaking change. You can also customize the activity bar icons in your extension, outside of Theia; so you do not force everyone else to adjust to your requirements. What do you think?

**Comment by daimor** [comment]
My extension is originally for VSCode. So, I just did the change to make it work exactly the same as in VSCode.
So, I don't use Theia API in extension, and I suppose as Theia working on support for VSCode extensions, it should be done on Theia side, instead of saying each extensions developer how they should fix their extension to fit some styling issues in Theia.

**Comment by daimor** [comment]
I have an idea, how to fix it better

**Comment by daimor** [comment]
@kittaakos could you review my PR #8505 with the fix? It should be universal now and work for any icon.

**Pull request merged**